### ISSUE TITLE
Navigation not visible properly

### ISSUE THREAD

**Issue by srishti023** [vishal02527/My-Strength-Shree-Krishna#358]
navigation bar is not displayed correctly on the Devotional Resources page, impacting user access to other sections of the website.
![Screenshot 2024-10-12 101829](https://github.com/user-attachments/assets/6d4e52b5-2233-4cb7-99d3-700ee1f2e7cc)
I want to solve this issue please assign me !
Thank you

**Comment by vishal02527** [comment]
@srishti023 Also check this problem in other pages also, if it is found then resolve them too.

**Comment by srishti023** [comment]
Yes, I will check for the same issue on other pages as well. Please assign me to this issue, and I’ll resolve them if found.
